### PR TITLE
add documentation for new websocket

### PR DESCRIPTION
**Pull request by samweinberg23**

### CODE REVIEW

File: docs/websocket.md [hunk 1]
```diff
@@ -586,3 +586,154 @@ eg:
...
+
+|Field Name|JSON type|Description|
+|----------|---------|-----------|
+|type|string|Must be set to "subscribe"|
+|channel|string|Must be set to "trades"|
+|id|string|The market to subscribe to. e.g. WETH-DAI, WETH-USDC, DAI-USDC|
+
+#### Initial Response
+
+The initial response will contain the entire state of the active market:
```
Comment (BrendanChou) [inline docs/websocket.md]: nit: I think the initial response will probably return the most recent 100 trades

File: docs/websocket.md [hunk 2]
```diff
@@ -586,3 +586,154 @@ eg:
...
+  "type": "unsubscribe",
+  "channel": "trades",
+  "id": "DAI-USDC"
+}
+```
+
+|Field Name|JSON type|Description|
+|----------|---------|-----------|
+|type|string|Must be set to "unsubscribe"|
+|channel|string|The channel to unsubscribe from|
```
Comment (BrendanChou) [inline docs/websocket.md]: nit: must be trades in this case probably. Anyway if this is common to all websockets, then don't need to repeat it here, just put it in an easy-to-access place in the websockets section

Comment (BrendanChou) [reply docs/websocket.md]: Same with `Subscribing` and `Response`. No need to have this in all sections

Comment (samweinberg23) [reply docs/websocket.md]: I think it's easier for readers to jump to a section and immediately see the fields for whatever they're trying to do. The line `|channel|string|The channel to unsubscribe from|` is used throughout websocket.md

**Pull request merged**